build.sh

```diff
@@ -15,17 +15,13 @@ for arg in "$@"; do
   esac
 done
 
-if [ -z "$SEQ" ] && SEQ=nyan.y4m && [ ! -f nyan.y4m ]; then
-  #SEQ10=nyan10.y4m
-  #SEQ12=nyan12.y4m
+if [ -z "$SEQ" ] && SEQ=small_input.y4m && [ ! -f small_input.y4m ]; then
 
-  if ! wget -nc https://mf4.xiph.org/~ltrudeau/videos/nyan.y4m &&
-    ! curl -fkLs https://mf4.xiph.org/~ltrudeau/videos/nyan.y4m -o nyan.y4m; then
+  if ! wget -nc https://github.com/xiph/rav1e/blob/master/tests/small_input.y4m &&
+    ! curl -fkLs https://github.com/xiph/rav1e/blob/master/tests/small_input.y4m -o nyan.y4m; then
```

Comment:
You should be getting from `/raw/` not `/blob/` (the latter gives you html). Also I think you want to `-o small_input.y4m` in your curl command.

Reply:
Given that this test sequence is in the same repo, why not skip downloading altogether? We can just set the default input to `tests/small_input.y4m`.

Reply:
Right, I shouldn't be writing PRs at night